packages/process-roles-directives/src/index.js

```diff
@@ -42,16 +54,16 @@ export function processRolesDirectives(
     /** @param {Node} node */
     function collectIdentifiers(node) {
         switch (node.type) {
-            case 'paragraph' || 'heading' || 'mystRole' || 'mystDirective':
+            case 'paragraph' || 'heading' || 'role' || 'mystDirective':
```

Comment:
Should be mystRole / mystDirective?

Reply:
Ideally, these names would be referenced by some central constant